### PR TITLE
Tier 1 open data links

### PR DESCRIPTION
**Pull request by steveoh**
i went through the tier 1 data sets from the stewardship doc and updated their links to point at open data. I'm not sure what to do with the package links any longer. I vote to remove those

### CODE REVIEW

File: data/boundaries/citycountystate/index.html [hunk 1]
```diff
@@ -165,8 +152,7 @@ <h5 id="CountyBoundaries-downloads">Downloads</h5>
     </div>
     <div class="panel-content">
       <ul class="dotless no-padding" id="CountyBoundaries-dlist">
-        <li><a href="https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mbnFpYUl1bDZtaDQ&export=download">County Boundaries: File Geodatabase</a></li>
-        <li><a href="https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mc3BGRXhKa0N0U2c&export=download">County Boundaries: Shapefile</a></li>
+        <li><a href="https://opendata.gis.utah.gov/datasets/utah-county-boundaries">County Boundaries: SGID Open Data Downloads and Services</a>
```
Comment (stdavis) [inline data/boundaries/citycountystate/index.html]: Don't we have a section for web services on the data pages as well? I wonder if this link should go there? Maybe in both download and web services? Just a thought.

Comment (steveoh) [reply data/boundaries/citycountystate/index.html]: yeah i thought that was redundant but am up for a mind change. I think the data pages could actually use a refresh with our new sgid. Let's chat about it at the next status meeting.

**Comment by steveoh** [comment]
I updated the city/county/state, parcels, and some of plss with the feedback from the meeting. I'll work on the rest of the tier1's tomorrow.

**Comment by eneemann** [comment]
I don't think I'm familiar enough with the code to go through all of the changes in a meaningful way.  However, I played around on the preview page and generally saw what I expected with link changes pointing to Open Data and direct downloads from Open Data, where applicable.  A couple of notes are below.  Some of these may not necessarily be in our control (or more of an AGOL thing):

Land Ownership
- Still has download button (points to SITLA Open Data)
- Not available as file geodatabase  in Open Data (SITLA's Open Data)

Address points
- Still has download button (points to Google Drive)
- Not available as file geodatabase in Open Data

**Comment by gregbunce** [comment]
nice work, this is begging to take shape.  

general comments:

Overall, I think the [Muni Boundary page](https://deploy-preview-1261--gis-utah-gov.netlify.com/data/boundaries/citycountystate/) is looking like a good template.  I like that the package downloads are no longer there.  and i like the reorganization of the download area with "Downloads and Web Services" consolidated into one.  

maybe it's just me, but I'm still not convinced that we need the direct download links on our AGRC data webpages, but, no harm.   I'm fine with just a link to Open Data page. 

It would be nice to see the same layout from the Muni Boundary page on all the data pages.

Should the links to open data, open in a new browser tab?  Seems like I always need to get back to gis.utah.gov/data for something.  but, then again, I have no fear of many tabs.  

Lastly, the "Web Application" link to AGOL at the top of the data page.  Should we replace this section with the title "Open Data" or "View Data" and have a quick link to the open data page there?  Seems like AGOL link should be removed.  

Again, this is looking good!  Thanks!

**Comment by steveoh** [comment]
> I don't think I'm familiar enough with the code to go through all of the changes in a meaningful way. 

@eneemann yeah I'm not expecting you to go through the code. I'm more interested in the results on the website preview.

**Comment by steveoh** [comment]
> Lastly, as Nathan noted... the "Web Application" link to AGOL at the top of the data page. Should we replace this section with the title "Open Data" or "View Data" and have a quick link to the open data page there? This might make Open Data stand out a bit more. But, at a minimum, it seems like the AGOL link should be removed.

The web application link typically goes to places where the data is highlighted; the raster app, the parcel app, the wri web app etc. I'm not sure why someone made an agol web map for this page. It wouldn't make sense to rename the title to open data or view data when most of the links actually go to web applications.

Thank you for your feedback. I will reorder the download links to put open data at the top.

**Comment by steveoh** [comment]
> Not available as file geodatabase in Open Data (SITLA's Open Data)

I reached out to Barry at SITLA to see if he would enable this.

> Not available as file geodatabase in Open Data

@stdavis has a script that _should_ have enabled this on every dataset. I'll go manually check the box now.

For future reference, this is the setting in the agol item page under settings. It's all the way at the bottom.

image

**Comment by mheagin** [comment]
Just to chime in here the AGOL maps / Web App views were created so non-GIS user, Recorders, City Planners  could look at and review their boundaries. Was the easiest option at the time.

**Comment by steveoh** [comment]
I think I hit all the tier 1's now. This removed the download link, the large page duplicate title, the package contents text, and added links to open data, file gdb, and shapefile.

**Comment by steveoh** [comment]
so everyone is happy with this and I should merge it? give a thumbs up or chat me in slack

**Pull request merged**